### PR TITLE
Fix markdown syntax error in README and introduction

### PR DESCRIPTION
**Pull request by Rhian**
This commit fixes a small typo in the Markdown syntax linking to the Contracts repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
...
+Welcome to Boson Protocol. In our [Contracts repository](https://github.com/bosonprotocol/contracts) you will find a set of Solidity smart contracts that implement the protocol. You are invited to learn more about the project through its code and perhaps test locally how you might use it within your own project. This is version 1 of the Protocol. You will find the addresses of the deployed contracts on Ethereum's main net and Ropsten test net [here](https://github.com/bosonprotocol/contracts/blob/main/docs/contracts/deployment.md).
 
 This version of Boson Protocol is a stepping stone on the way to the release of version 2 of the Protocol next year, which will be accompanied with SDKs and plug-ins to make it easy to integrate. While you may prefer to wait until the new features that will be released in version 2 before building dApps using the Protocol, we encourage you to use the contracts locally and to participate in our [bug bounty](/contributing/bug-bounty.md).
```
Comment (hswopeams) [inline README.md:8]: The bug bounty link does not work for me

File: src/docs/introduction.md [hunk 2]
```diff
@@ -10,13 +10,13 @@ redirect_from:
...
+Welcome to Boson Protocol. In our
+[Contracts repository](https://github.com/bosonprotocol/contracts) you will find
+a set of Solidity smart contracts that implement the protocol. You are invited
+to learn more about the project through its code and perhaps test locally how
+you might use it within your own project. This is version 1 of the Protocol. You
+will find the addresses of the deployed contracts on Ethereum's main net and
+Ropsten test net
 [here](https://github.com/bosonprotocol/contracts/blob/main/docs/contracts/deployment.md).
 
 This version of Boson Protocol is a stepping stone on the way to the release of
```
Comment (hswopeams) [inline src/docs/introduction.md:22]: The bug bounty link also does not work for me on this page.

**Review by hswopeams — CHANGES_REQUESTED** [review]
See comments about bug bounty links.

File: README.md (hunk 1 above)
Comment (hswopeams) [reply README.md:8]: The link works now.

File: src/docs/introduction.md (hunk 2 above)
Comment (hswopeams) [reply src/docs/introduction.md:22]: The link works now.

**Review by hswopeams — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Rhian) [reply README.md:8]: Have updated to use the link to the community repo, to keep it in line with the `contracts` repo for now

File: src/docs/introduction.md (hunk 2 above)
Comment (Rhian) [reply src/docs/introduction.md:22]: Have updated to use the link to the community repo, to keep it in line with the `contracts` repo for now, as above

**Review by lauraoconnor-gh — APPROVED** [review]

**Pull request merged**